src/CMakeLists.txt

```diff
@@ -3,24 +3,52 @@ include_directories(${PROJECT_SOURCE_DIR}/headers/algorithms)
 include_directories(${PROJECT_SOURCE_DIR}/headers/python_bindings)
 include_directories(${PROJECT_SOURCE_DIR}/headers/carma/include)
 include_directories(${PROJECT_SOURCE_DIR}/headers/carma/include/carma_bits)
+# Include directory for windows
+if(WIN32)
+include_directories(${PROJECT_SOURCE_DIR}/headers/armadillo/include)
+include_directories(
+        ${PROJECT_SOURCE_DIR}/headers/armadillo/include/armadillo_bits)
+set(OPENBLAS_dir ${PROJECT_SOURCE_DIR}/headers/armadillo/examples/lib_win64)
+endif()
 
-# For Mac Github Runner to find OpenMP -- potentially unnecessary after fixing -Xpreprocessor -fopenmp issue
+# For Mac Github Runner to find OpenMP -- potentially unnecessary after fixing
+# -Xpreprocessor -fopenmp issue
+if(APPLE OR UNIX)
 include_directories(/usr/local/Cellar/libomp/15.0.2/include)
 include_directories(/usr/local/Cellar/libomp/15.0.7/include)
 include_directories(/usr/local/Cellar/libomp)
+include_directories(/usr/local/opt/libomp/include)
 include_directories(/usr/local/opt/libomp/lib)
 include_directories(/usr/local/opt/libomp)
 include_directories(/usr/local/opt)
+endif()
 
-add_executable(BanditPAM main.cpp)
+if(WIN32)
+add_executable(BanditPAM getopt.cpp main.cpp )
+
```

Comment:
This is necessary for the CMake build. Otherwise, we get this:

```
3>------ Build started: Project: BanditPAM, Configuration: Release x64 ------
3>Building Custom Rule C:/impt/CLionTest/BanditPAM/src/CMakeLists.txt
3>cl : command line  warning D9002: ignoring unknown option '-O3'
3>main.cpp
3>main.obj : error LNK2019: unresolved external symbol getopt referenced in function main
3>main.obj : error LNK2019: unresolved external symbol optarg referenced in function main
3>main.obj : error LNK2019: unresolved external symbol optind referenced in function main
3>main.obj : error LNK2019: unresolved external symbol optopt referenced in function main
3>C:\impt\CLionTest\BanditPAM\build\src\Release\BanditPAM.exe : fatal error LNK1120: 4 unresolved externals
3>Done building project "BanditPAM.vcxproj" -- FAILED.
4>------ Skipped Build: Project: ALL_BUILD, Configuration: Release x64 ------
4>Project not selected to build for this solution configuration
========== Build: 2 succeeded, 1 failed, 0 up-to-date, 1 skipped ==========
========== Elapsed 00:11.257 ==========
.\BanditPAM : The term '.\BanditPAM' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\adars\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:11 char:43
+ function fresh { rebuild; cd src/Release; .\BanditPAM -f ../../../dat ...
+                                           ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (.\BanditPAM:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```